plugins/de.cau.cs.kieler.klighd/src/de/cau/cs/kieler/klighd/internal/macrolayout/KlighdDiagramLayoutConnector.java

```diff
@@ -672,15 +673,15 @@ private void applyLayout(final LayoutMapping mapping, final boolean suppressEdge
                 public Boolean caseElkNode(final ElkNode layoutNode) {
                     final KNode node = (KNode) element;
                     
-                    // Get properties that shall be preserved from ElkGraph to KGraph
-                    List<IProperty<?>> propertiesToPreserve = KlighdDataManager.getInstance().getPreservedProperties();
-                    
-                    // Preserve properties
-                    for (IProperty<?> property : propertiesToPreserve) {
-                        if (layoutNode.hasProperty(property)) {
-                            node.setProperty((IProperty<Object>) property, (Object) layoutNode.getProperty(property));
-                        }
-                    }                  
+//                    // Get properties that shall be preserved from ElkGraph to KGraph
+//                    List<IProperty<?>> propertiesToPreserve = KlighdDataManager.getInstance().getPreservedProperties();
+//                    
+//                    // Preserve properties
+//                    for (IProperty<?> property : propertiesToPreserve) {
```

Comment:
This is the relevant code

Reply:
So it is the case that there is special handling for some ELK properties to be preserved. In general afaik ELK usually only outputs the final layout of the graph and all properties used by ELK to calculate that stay internal to ELK as they are of no further use for KLighD and its renderings. The interactive layout somewhat broke this default usage as for these interactive properties we do care about them later, especially in the rendering, to correctly display the ELK-internal layers to the user.
So I would guess for that to work we still either need this generic preservation of ELK properties in this ElkGraph->KGraph layout mapping step or only copy them over individually here as done in the initial commit where this popped up: https://github.com/kieler/KLighD/commit/c88c6862c5182dc7eefbe8535ac2e69ae04bb391
As this is a single case where this is used now, I would be fine with either way.